app/client/src/components/viewer/ol/Map.vue

```diff
@@ -305,6 +305,21 @@ export default {
     EventBus.$emit("ol-map-mounted", me.map);
     //Add map to the vuex store.
     // resize the map, so it fits to parent
+
+    me.map.on("singleclick", function(evt) {
+      const viewResolution = me.map.getView().getResolution();
+      console.log("was geht's digga", evt, viewResolution);
```

Comment:
:) nice console log. But I would also remove it.

Reply:
😂 yeah forgot that one